src/browser/api_protocol/api_handlers/system.js

```diff
@@ -41,6 +41,7 @@ function SystemApiHandler() {
         'get-el-ipc-config': getElIPCConfig,
         'get-environment-variable': { apiFunc: getEnvironmentVariable, apiPath: '.getEnvironmentVariable' },
         'get-host-specs': { apiFunc: getHostSpecs, apiPath: '.getHostSpecs' },
+        'get-min-log-level': getMinLogLevel,
```

Comment:
Not to confuse the issue, but IMO neither terminology (min vs max) is appropriate here.

Compare to Java _log4j_:

```
log.setLevel(Level.WARN);
```

Or MongoDB:

```
db.setLogLevel(<level>, <component>)
```

Googling `'setMinLogLevel'` finds that Chromium is the only major project to use this terminology. Meanwhile `'setMaxLogLevel'` appears to have no common usage.